templates/web/smidsy/footer_extra_js.html

```diff
@@ -1,4 +1,3 @@
 [% scripts.push(
-    version('/jslib/jquery-1.7.2.min.js'),
```

Comment:
There isn't one in core on e.g. the front page. Which is why it's here. It's only ever output once.

Reply:
…But smidsy doesn’t actually _use_ jQuery on the homepage, does it?

Reply:
Oh! I see. `report.js` does, even if none of the selectors it uses actually match anything.